src/Domain/File.php

```diff
@@ -123,6 +105,20 @@ public function getFileInfo(): SplFileInfo
         return $this->fileInfo;
     }
 
+    /**
+     * Active fix mode. It's used to prevent report twice
+     * details because fixer relaunch process method.
+     */
+    public function activeFix(): void
```

Comment:
Consider naming this `enableFix`, as the counterpart is called `disableFix`

Reply:
Done :wink: